Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -0,0 +1,1249 @@
+            'Id': attachment_entry.get('id', ''),
+            'Attachment Name': attachment_entry.get('filename', ''),
+        }
+        attachment_dict |= {'Issue Id': issue_id_or_key} if is_id else {'Issue Key': issue_id_or_key}
+        markdown_dict.append(attachment_dict)
+    return CommandResults(
+        readable_output=tableToMarkdown('Attachment added successfully', markdown_dict)
+    )
+
+
+def issue_get_attachment_command(client: JiraBaseClient, args: Dict[str, str]) -> List[Dict[str, Any]]:
+    """Get attachments content
+
+    Returns:
+        Dict[str, Any]: A dictionary the represents a file entry to be returned to the user
+    """
+    attachments_id = argToList(args.get('id', ''))
+    files_result: List[Dict[str, Any]] = []
+    for attachment_id in attachments_id:
+        files_result.append(create_file_info_from_attachment(client=client, attachment_id=attachment_id))
+    return files_result
+
+
+def get_specific_fields_command(client: JiraBaseClient, args: Dict[str, str]) -> CommandResults:
+    issue_id_or_key = args.get('issue_id', '') if args.get('issue_id', '') else args.get('issue_key', '')
+    if not issue_id_or_key:
+        return_error(ID_OR_KEY_MISSING_ERROR)
+    fields = argToList(args.get('fields', ''))
+    # if fields:
+    #     fields = set(field.lower() for field in fields)
+    res = client.get_issue(issue_id_or_key=issue_id_or_key)
+    # Get the general fields of the issue
+    markdown_dict, outputs = create_issue_md_and_outputs(client=client, res=res)
+    custom_id_name_mapping, custom_fields_res = get_custom_fields_mapping(client=client)
+    for field in fields:
+        readable_field_name = custom_id_name_mapping.get(field, '')
+        if readable_field_name:
+            titled_field = readable_field_name.title().replace(' ', '')
+            markdown_dict[titled_field] = outputs[titled_field] = demisto.get(res, f'fields.{field}', '')
+        else:
+            return_warning(f'The field {field} was not found.')
+    return CommandResults(
+        outputs_prefix='Ticket',
+        outputs=outputs,
+        outputs_key_field='Id',
+        readable_output=tableToMarkdown(name=f'Issue {outputs.get("Key", "")}', t=markdown_dict,
+                                        headerTransform=pascalToSpace),
+        raw_response=[res, custom_fields_res]
+    )
+
+
+# User Commands
+def get_id_by_attribute_command(client: JiraBaseClient, args: Dict[str, str]) -> CommandResults:
+    attribute = args.get('attribute', '')
+    max_results = arg_to_number(args.get('max_results', None))
+    if not max_results:
+        max_results = 50
+    res = client.get_id_by_attribute(attribute=attribute, max_results=max_results)
+    if not res:
+        return CommandResults(readable_output=f'No Account ID was found for attribute: {attribute}.')
+    outputs = {'Attribute': attribute}
+    is_jira_cloud = isinstance(client, JiraCloudClient)
+    account_ids: List[str] = []
+
+    if len(res) == 1:
```

Comment:
Since we compare the given attribute to the email address in order to retrieve the account id, and the email address can be empty for privacy reasons, we want to avoid a situation where we actually receive a valid user, but since the email address can be empty, we will output to the user than no user was found, even though we got a valid user in return.